modules/ROOT/pages/kubernetes/operations/backup-restore.adoc

```diff
@@ -11,7 +11,8 @@ For more information, see xref:kubernetes/accessing-neo4j.adoc[Accessing Neo4j].
 [[kubernetes-neo4j-backup-cloud]]
 == Back up a database to a cloud provider (AWS, GCP, and Azure) bucket
```

Comment:
Backup databases ?

Reply:
I also thought so, but since this is the case from 4.4.22, I decided to leave it as it is and add a note.  Do you think I should change it?